common/translations/lang/en.json

```diff
@@ -253,7 +253,7 @@
     "DEP_interface": "Generated Interface ",
     "NODE_Title": "Set Up Your Custom Node",
     "NODE_Subtitle": "To connect to a local node...",
-    "NODE_Warning": "Your node must be HTTPS in order to connect to it via MyCrypto.com. You can [download the MyCrypto repo & run it locally](https://github.com/MyCryptoHQ/MyCrypto/releases/latest) to connect to any node. Or, get free SSL certificate via [LetsEncrypt](https://letsencrypt.org/)",
+    "NODE_Warning": "Your node must be HTTPS in order to connect to it via MyCrypto.com. You can [download the MyCrypto repo & run it locally](https://github.com/MyCryptoHQ/mycrypto.com/releases/latest) to connect to any node. Or, get free SSL certificate via [LetsEncrypt](https://letsencrypt.org/)",
```

Comment:
We want to keep these URLs like this, even if they're not active right now. This repo will replace mycrypto.com when it launches.